apps/advisor/pkg/app/checks/datasourcecheck/check.go

```diff
@@ -83,12 +83,18 @@ func (s *uidValidationStep) Run(ctx context.Context, obj *advisor.CheckSpec, i a
 	// Data source UID validation
 	err := util.ValidateUID(ds.UID)
 	if err != nil {
+		icon := "document-info"
```

Comment:
Is this a CSS class in the backend? 🙈

Reply:
that's what you get when you return pseudo HTML from the backend 😅